README.md

```diff
@@ -46,6 +51,7 @@ Please be respectful of the people who have created these resources. If you want
     * [CHME7340 Chemical Engineering Kinetics and Reactor Design (Grad)](https://github.com/rwest/CHME7340) - Northeastern University, Prof. Richard West.
     * [06-364 Chemical and Reaction Engineering](https://github.com/jkitchin/s17-06364) - Carnegie Mellon University, Prof. John Kitchin, Spring 2017.
     * [06-625 Chemical and Reactive Systems](https://github.com/jkitchin/f16-06625) - Carnegie Mellon University, Prof. John Kitchin, Fall 2016. ([Fall 2015 version](https://github.com/jkitchin/f15-06625) also available).
+    * [Reactor design - Machine Learning approch](https://github.com/jeff-ball/Machine-Learning-Chemical-Engineering-Approach) - AUB, Lebanon
```

Comment:
same as the other 2 comments in this section. Maybe we should reorganize the other sections to have subheadings for each core class like this section has? So under "Software", for example, it could group the softwares under headings like "Process Control", "Chemical Kinetics & Reactor Engineering/Design", etc. Otherwise the software section may become too long. What do you think?

Reply:
yeah that's fair will change it